rllib/algorithms/dqn/dqn_rainbow_catalog.py

```diff
@@ -0,0 +1,191 @@
+                ),
+                std_init=self.std_init,
+            )
+            self.vf_head_config = NoisyMLPHeadConfig(
+                input_dims=self.latent_dims,
+                hidden_layer_dims=self.af_and_vf_head_hiddens,
+                hidden_layer_activation=self.af_and_vf_head_activation,
+                output_layer_activation="linear",
+                output_layer_dim=1,
+                std_init=self.std_init,
+            )
+        else:
+            self.af_head_config = MLPHeadConfig(
+                input_dims=self.latent_dims,
+                hidden_layer_dims=self.af_and_vf_head_hiddens,
+                hidden_layer_activation=self.af_and_vf_head_activation,
+                output_layer_activation="linear",
+                output_layer_dim=int(
+                    action_space.n * self._model_config_dict["num_atoms"]
+                ),
+            )
+            self.vf_head_config = MLPHeadConfig(
+                input_dims=self.latent_dims,
+                hidden_layer_dims=self.af_and_vf_head_hiddens,
+                hidden_layer_activation=self.af_and_vf_head_activation,
+                output_layer_activation="linear",
+                output_layer_dim=1,
+            )
+
+    @OverrideToImplementCustomLogic
+    def build_af_head(self, framework: str) -> Model:
+        """Build the A/Q-function head.
+
+        Note, if no dueling architecture is chosen, this will
+        be the Q-function head.
+
+        The default behavior is to build the head from the `af_head_config`.
+        This can be overridden to build a custom policy head as a means to
+        configure the behavior of a `DQNRainbowRLModule` implementation.
+
+        Args:
+            framework: The framework to use. Either "torch" or "tf2".
+
+        Returns:
+            The advantage head in case a dueling architecutre is chosen or
+            the Q-function head in the other case.
+        """
+        return self.af_head_config.build(framework=framework)
+
+    @OverrideToImplementCustomLogic
+    def build_vf_head(self, framework: str) -> Model:
+        """Build the value function head.
+
+        Note, this function is only called in case of a dueling architecture.
+
+        The default behavior is to build the head from the `vf_head_config`.
+        This can be overridden to build a custom policy head as a means to
+        configure the behavior of a `DQNRainbowRLModule` implementation.
+
+        Args:
+            framework: The framework to use. Either "torch" or "tf2".
+
+        Returns:
+            The value function head.
+        """
+
+        return self.vf_head_config.build(framework=framework)
+
+    @override(Catalog)
+    def get_action_dist_cls(self, framework: str) -> "TorchCategorical":
+        # We only implement DQN Rainbow for Torch.
+        assert framework == "torch"
+        return TorchCategorical
+
+
+# __sphinx_doc_end__
```

Comment:
nit: remove these __sphinx tags here (and on top) as we don't refer to these files right now from the docs.